CHANGELOG.md

```diff
@@ -5,22 +5,143 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+|                               | `--skip_consensus`                    |
+|                               | `--skip_ivar_trim`                    |
+|                               | `--skip_nanoplot`                     |
+|                               | `--skip_pangolin`                     |
+|                               | `--skip_pycoqc`                       |
+|                               | `--skip_sra_fastq_download`           |
+|                               | `--spades_hmm`                        |
+|                               | `--spades_mode`                       |
+| `--cut_mean_quality`          |                                       |
+| `--filter_unmapped`           |                                       |
+| `--ivar_trim_min_len`         |                                       |
+| `--ivar_trim_min_qual`        |                                       |
+| `--ivar_trim_window_width`    |                                       |
+| `--kraken2_use_ftp`           |                                       |
+| `--max_allele_freq`           |                                       |
+| `--min_allele_freq`           |                                       |
+| `--min_base_qual`             |                                       |
+| `--min_coverage`              |                                       |
+| `--min_trim_length`           |                                       |
+| `--minia_kmer`                |                                       |
+| `--mpileup_depth`             |                                       |
+| `--name`                      |                                       |
+| `--qualified_quality_phred`   |                                       |
+| `--save_align_intermeds`      |                                       |
+| `--save_kraken2_fastq`        |                                       |
+| `--save_sra_fastq`            |                                       |
+| `--skip_sra`                  |                                       |
+| `--skip_vg`                   |                                       |
+| `--unqualified_percent_limit` |                                       |
+| `--varscan2_strand_filter`    |                                       |
+
+> **NB:** Parameter has been __updated__ if both old and new parameter information is present.  
+> **NB:** Parameter has been __added__ if just the new parameter information is present.  
+> **NB:** Parameter has been __removed__ if new parameter information isn't present.
+
+### Software dependencies
+
+Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
```

Comment:
Interesting ... does this mean that we cannot pin nf-core-modules to a specific release version or git commit when running this workflow?

Reply:
Nope, it means that we can have different modules that use different versions of tools e.g. we could have a container used for `BWA` alignment as [here[(https://github.com/nf-core/modules/blob/0218824b860979b7c62bcf3159c7288aa222861a/software/bwa/mem/main.nf#L14) that use a different version of SAMtools then say `samtools sort` as defined [here](https://github.com/nf-core/modules/blob/0218824b860979b7c62bcf3159c7288aa222861a/software/samtools/sort/main.nf#L14). Since the modules are self-contained this will be expected.

If you want to pin to a particular version, that version needs to at the very least have existed on nf-core/modules. Then it will be possible.

Reply:
Ah I see, thanks.  In terms of reproducibility for a specific run, does a run against a specific commit or release version of viralrecon also "pin" nf-core/modules?  Sorry, I haven't really thought much about how DSL2 works in these terms.